sound/soc/sof/intel/hda.h

```diff
@@ -454,6 +454,8 @@
 #define SSP_SET_SFRM_CONSUMER	BIT(24)
 #define SSP_SET_CBP_CFP		(SSP_SET_SCLK_CONSUMER | SSP_SET_SFRM_CONSUMER)
 
+#define HDA_EXT_ADDR		0
+#define HDA_EXT_CODEC(x) ((x) & BIT(HDA_EXT_ADDR))
```

Comment:
I guess address 0 is always for external codecs and BIT(1) and BIT(3) is not used?

Reply:
In theory, a dual HDA codec system would use addr1 (SDI1), but in practise we've never had these with SOF. In most systems, codec count is limited to 3 with address 2 hardcodec to PCH display codec. For purpose of this patch (and check), this is ok.

Reply:
Yeah, I thought about the SDI1 case but we have no way to test it, so I chose to limit the scope to BIT(0).